virt/release_notes/virt-4-18-release-notes.adoc

```diff
@@ -211,6 +211,10 @@ include::snippets/technology-preview.adoc[]
 * {VirtProductName} links a service account token in use by a pod to that specific pod. {VirtProductName} implements a service account volume by creating a disk image that contains a token. If you migrate a VM, then the service account volume becomes invalid. (link:https://issues.redhat.com/browse/CNV-33835[*CNV-33835*])
 ** As a workaround, use user accounts rather than service accounts because user account tokens are not bound to a specific pod.
 
+//CNV-56998 is doc issue; CNV-56659 is original issue
+* Virtual machine instance migrations might fail during workload updates or when a large number of migrations are triggered. This is more likely to occur if there are at least 400 pending or undefined migrations. (link:https://issues.redhat.com/browse/CNV-56659[*CNV-56659*])
```

Comment:
> pending or undefined

Should be:
pending and/or undefined

Reply:
I attempted to reword this without using "and/or," because it's not allowed in our style guide[1]. Let me know if it sounds better to you: "This is more likely to occur if there are at least 400 pending migrations, undefined migrations, or a combination of both."

[1]

> Do not use **and/or** conjunction
> Do not use. Depending on the context, use one of the following constructions: a and b, a or b, or a, b, or both.
> 
> The construction and/or can be ambiguous and causes translation problems.

Reply:
:+1:  LGTM!